plugins/out_datadog/datadog.c

```diff
@@ -581,7 +581,10 @@ static struct flb_config_map config_map[] = {
      0, FLB_TRUE, offsetof(struct flb_out_datadog, json_date_key),
      "Date key name for output."
     },
-
+    {
+    FLB_CONFIG_MAP_STR, "site", NULL, 0, FLB_FALSE, offsetof(struct flb_out_datadog, site),
+    "DataDog site for telemetry data (e.g., 'datadoghq.eu', 'datadoghq.com', 'us3.datadoghq.com'). The plugin will construct the full hostname by prepending 'http-intake.logs.' to this value."
```

Comment:
line is too long, up to 120 is fine (from the source code perspective), just continue the string in another line

Reply:
if you set `FLB_TRUE` in the 5th parameter, the value will be written to context->site